tests/integration/pkg/resource/manager.go

```diff
@@ -17,6 +17,9 @@ import (
 	"k8s.io/client-go/restmapper"
 )
 
+// TestGatewayOperatorName contains name for APIGateway operator resource
```

Comment:
Having the `TestGatewayOperatorName` in this package feels wrong. What's the reason why it's here?

Reply:
It's used with manager functions, and should be accessible everywhere (both in test scenarios as well as in `main_test.go`). Any suggestion where to move it?

Reply:
I think it should be moved to main_test, since this is the only correct usage. It's used in the gateway testsuite for now, but once this is correctly implemented and will apply the APIGateway, the name of the gateway should be part of the creation step and therefore this doesn't need to be referenced. 
Also the code in main_test.go should be moved to a separate file in the future, since it'S only relevant for Before and After testsuite code. But this will be done in a later implementation as far as I understood..

Reply:
Also the name of the variable is not correct. It's not the operator, it is the custom resource.

Reply:
Moved to `config.go`